bin/pre-commit-hook.js

```diff
@@ -3,28 +3,78 @@
 const execSync = require( 'child_process' ).execSync;
 const spawnSync = require( 'child_process' ).spawnSync;
 const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const prettier = require( 'prettier' );
+const path = require( 'path' );
 
-console.log( '\nBy contributing to this project, you license the materials you contribute ' +
+console.log(
+	'\nBy contributing to this project, you license the materials you contribute ' +
 	'under the GNU General Public License v2 (or later). All materials must have ' +
-	'GPLv2 compatible licenses — see .github/CONTRIBUTING.md for details.\n\n' );
+	'GPLv2 compatible licenses — see .github/CONTRIBUTING.md for details.\n\n'
+);
 
 // Make quick pass over config files on every change
 require( '../server/config/validate-config-keys' );
 
 const files = execSync( 'git diff --cached --name-only' )
```

Comment:
We often limit this to certain operations. Probably not needed, for example, for deleted or renamed files…